### PR TITLE
Headmethod

### PR DESCRIPTION
**Pull request by jeffrey20122021**

### CODE REVIEW

File: test/GithubApi.Redirect.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+
+chai.use(chaiSubset);
+const object = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+it('test', async () => {
```
Comment (holgiosalos) [inline test/GithubApi.Redirect.test.js]: you need to add an additional `it` to verify the redirect is also present when you do a call with the head method, and the two `it`s should go within a `describe`

**Review by holgiosalos — COMMENTED** [review]
- test/GithubApi.Put.test.js should not be parth of this PR

File: test/GithubApi.Redirect.test.js (hunk 1 above)
Comment (jeffrey20122021) [reply test/GithubApi.Redirect.test.js]: done @holgiosalos

**Comment by jeffrey20122021** [comment]
> * test/GithubApi.Put.test.js should not be parth of this PR

how i can eliminate it?
can use this button?
image

**Comment by holgiosalos** [comment]
@jeffrey20122021 no, you just need to merge main/master with this branch and push the merge commit

i merge correctly @holgiosalos

File: test/GithubApi.Redirect.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  describe('head', async () => {
+    it('HEAD redirection', async () => {
+      const response = await object.head('https://github.com/aperdomob/redirect-test');
+      expect(response.request.path).to.equal('/aperdomob/new-redirect-test');
+    });
+    it('GET redirection', async () => {
+      const response = await object.get('https://github.com/aperdomob/redirect-test');
+      expect(response.data).to.contains('href="https://github.com/aperdomob/new-redirect-test"');
+      expect(response.request.path).to.equal('/aperdomob/new-redirect-test');
+    }).timeout(10000000);
```
Comment (holgiosalos) [inline test/GithubApi.Redirect.test.js]: is this timeout required?

Comment (jeffrey20122021) [reply test/GithubApi.Redirect.test.js]: is not require , i delete it. @holgiosalos

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**